Protocol.md

```diff
@@ -187,8 +209,15 @@ The receiver:
     - MAY reject the contract.
   - if `cet_adaptor_signatures` or `refund_signature` fail validation:
     - MUST reject the contract.
-- if `funding_inputs` do not contribute at least `total_collateral_satoshis` plus [fee payment](Transactions.md#fee-payment)
-  - MUST reject the contract.
+  - if `funding_inputs` do not contribute at least `total_collateral_satoshis` plus [fee payment](Transactions.md#fee-payment)
+    - MUST reject the contract.
+
+The receiving node MUST reject the contract if:
+
+  - Any `input_serial_id` is duplicated
+  - Any `change_serial_id` is duplicated
+  - Any `change_serial_id` is equal to the `fund_output_serial_id`
+  - Any `payout_serial_id` is duplicated
```

Comment:
nit: it's a bit weird to me that these are split in two, is there a reason for that?